code/game/machinery/cloning.dm

```diff
@@ -301,10 +301,10 @@
 			src.occupant.adjustBrainLoss(-((speed_coeff/2)))
 
 			//So clones don't die of oxyloss in a running pod.
-			if (src.occupant.reagents.get_reagent_amount("inaprovaline") < 30)
-				src.occupant.reagents.add_reagent("inaprovaline", 60)
+			if (src.occupant.reagents.get_reagent_amount("perfluorodecalin") < 5)
+				src.occupant.reagents.add_reagent("perfluorodecalin", 5)

```

Comment:
People are gonna pop out of the cloning pods unable to talk, do epi or salbutamol